packages/clay-panel/src/index.tsx

```diff
@@ -91,10 +93,13 @@ const ClayPanel: React.FunctionComponent<IProps> & {
 		>
 			{!collapsable && (
 				<>
-					{displayTitle && (
+					{typeof displayTitle === 'string' ||
```

Comment:
I think react offers a helper function called `isReactElement`, so instead of checking for string/number you could do something like

```jsx
{React.isValidElement(displayTitle) ? (
	displayTitle
) : (
	<ClayPanelHeader>
		<span className="panel-title">{displayTitle}</span>
	</ClayPanelHeader>
)}
```

I _think_ this should work...

Reply:
It's [React.isValidElement](https://reactjs.org/docs/react-api.html#isvalidelement)